fe/fe-core/src/main/java/com/starrocks/qe/ConnectProcessor.java

```diff
@@ -332,6 +333,13 @@ protected void handleQuery() {
                         ErrorReport.reportAnalysisException(ErrorCode.ERR_UNSUPPORTED_PS, ErrorType.UNSUPPORTED);
                     }
                 }
+                hasQuery = parsedStmt instanceof QueryStatement;
```

Comment:
Is `INSERT INTO` QueryStatement?
If it is not, i think we need to make users can cancel it too.
I would like to suggest that when query id is given, this customer_query_id should be applied.

Reply:
Yes, `INSERT INTO` is QueryStatement.

We can focus on `Query` now. 
We have specified way to cancel other statement, like  `CANCEL LOAD`  by label for a load task.